src/js/views/patients/schedule/schedule_views.js

```diff
@@ -230,17 +277,44 @@ const ScheduleListView = CollectionView.extend({
       state: this.state,
     };
   },
-  emptyView: EmptyView,
+  childViewEvents: {
+    'filter'() {
```

Comment:
this can just be `'filter': 'debounceChildFilter'`.. err I guess a `render:children` version